model/testing/src/icon4py/model/testing/fixtures/datatest.py

```diff
@@ -61,7 +61,10 @@ def backend(request: pytest.FixtureRequest) -> gtx_backend.Backend:
 
 
 @pytest.fixture
-def experiment():
+def experiment() -> str:  # TODO(havogt): return definitions.Experiments.MCH_CH_R04B09
```

Comment:
should not this one become the parametrized fixture that you define in some test files directly. 

Even now it gets overwritten by parametrization in the tests and it is probably only here because some other fixture depends on it and/or because I thought in the beginning that you need a fixture in order to overwrite it.